lib/cassette/client.rb

```diff
@@ -20,12 +20,19 @@ def health_check
 
     def tgt(usr, pwd, force = false)
       logger.info 'Requesting TGT'
-      cache.fetch_tgt(force: force) do
+      cached = true
+
+      tgt = cache.fetch_tgt(force: force) do
+        cached = false
         response = http.post(tickets_path, username: usr, password: pwd)
         tgt = Regexp.last_match(1) if response.headers['Location'] =~ /tickets\/(.*)/
         logger.info "TGT is #{tgt}"
         tgt
       end
+
+      logger.info("TGT was cached with #{tgt}") if cached
```

Comment:
The usual terminology is cache hit/cache miss, what do you think about using it? Maybe we can merge with the message that logs the tgt

Reply:
this way?

```ruby
"TGT cache hit, value: '#{tgt}'"
```

Reply:
Exactly like that. We need to change the other one inside the block to say it is a miss

Reply:
Codeclimate did not like the ABC of that method 🙄

Reply:
I did a small refactoring to resolve the ABC problem, now I think it's done! Could you review again pls?